cmd/helm/installer/uninstall.go

```diff
@@ -21,13 +21,12 @@ import (
 
 	"github.com/ghodss/yaml"
 
+	"k8s.io/helm/pkg/kube"
 	"k8s.io/kubernetes/pkg/api"
 	kerrors "k8s.io/kubernetes/pkg/api/errors"
 	"k8s.io/kubernetes/pkg/apis/extensions"
 	"k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
 	"k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/typed/core/internalversion"
-
-	"k8s.io/helm/pkg/kube"
```

Comment:
Indeed, it seems like they enjoy it, still the precedence for packages being sorted in this order has always been implied. See [cmd/helm/init.go[(https://github.com/kubernetes/helm/blob/master/cmd/helm/init.go) (and anywhere else in the source code) for an example. It's just best practices we've been following.